src/gui/widgets/EffectChooser.h

```diff
@@ -102,11 +102,13 @@ struct EffectChooser : public juce::Component, public WidgetBaseMixin<EffectChoo
         case fxb_no_fx:
             return true;
         case fxb_no_sends:
-            if ((fxslot == 4) || (fxslot == 5))
+            if ((fxslot == fxslot_send1) || (fxslot == fxslot_send2) || (fxslot == fxslot_send3) ||
+                (fxslot == fxslot_send4))
                 return true;
             break;
         case fxb_scene_fx_only:
-            if (fxslot > 3)
+            if ((fxslot >= fxslot_send1 && fxslot == fxslot_global2) ||
```

Comment:
Did you really mean al == in the second?
This will only be true for fxslot == global2 or global 4
I think you mean >= fxslot_send1 && <= fxslot_global2

Reply:
yeah you definitely meant <=. Pushed a fix to your branch and will merge. Good catch!

Reply:
Yes that was an unfortunate typo, I did mean to cover the range so <= was definitely needed there 🙂